packages/volto/Makefile

```diff
@@ -72,10 +72,10 @@ install: ## Set up development environment
 cypress-install: ## Install Cypress for acceptance tests
 	$(NODEBIN)/cypress install
 
-../registry/dist: ../registry/src
+../registry/dist: $(shell find ../registry/src -type f)
 	(cd ../../ && pnpm build:registry)
 
-../components/dist: ../components/src
+../components/dist: $(shell find ../components/src -type f)
```

Comment:
This target is not invoked elsewhere in this Makefile or any other that I could find. Is that intentional or an oversight?

Reply:
Good eye! It is not used (yet) by core! but some projects can invoke it in case they already are using `@plone/components` as a dependency, so it's already there for them to use.